apps/sms/js/thread_list_ui.js

```diff
@@ -646,7 +648,7 @@ var ThreadListUI = {
     Settings.setReadAheadThreadRetrieval(this.FIRST_PANEL_THREAD_COUNT);
   },
 
-  renderThreads: function thlui_renderThreads(firstViewDoneCb, allDoneCb) {
+  renderThreads: function thlui_renderThreads(firstViewDoneCb) {
```

Comment:
Thanks for the link, will take a look! I'm ok with 2 promises, why not, and agree that it's better to have separate patch for the change and keep this v2.2 blocker patch simpler if possible.